apps/dashboard/src/components/workflow-editor/steps/base/base-body.tsx

```diff
@@ -24,7 +24,7 @@ export const BaseBody = () => {
           <FormControl>
             <InputRoot hasError={!!fieldState.error}>
               <ControlInput
-                className="h-[7.75rem]"
+                className="min-h-[7rem]"
```

Comment:
I copied the fix that Pawel did in the in-app body to protect the outline of the editor from breaking as new lines are added to code-mirror